tests/functional/cylc-combination-scripts/vr_workflow/flow.cylc

```diff
@@ -0,0 +1,8 @@
+[scheduling]
+    initial cycle point = 1500
+    [[graph]]
+        P1Y = foo
+
+[runtime]
+    [[foo]]
+        script = sleep 500
```

Comment:
Best to avoid arbitrary sleep.

If you want the workflow to stay running so you can try running `cylc vr` on it either:
* Start the workflow with `--pause` (avoids actually running the workflow).
* Have a task in the workflow run the `cylc vr` command (avoids timing issues).